src/vs/workbench/services/configuration/browser/configurationService.ts

```diff
@@ -1360,7 +1360,11 @@ class UpdateExperimentalSettingsDefaults extends Disposable implements IWorkbenc
 		const allProperties = this.configurationRegistry.getConfigurationProperties();
 		for (const property of properties) {
 			const schema = allProperties[property];
-			if (!schema?.tags?.includes('experimental')) {
+			const tags = schema?.tags;
+			// Many experimental settings refer to in-development or unstable settings.
+			// onexperiment more clearly indicates that the setting could be
+			// part of an experiment.
+			if (!tags || (!tags.includes('experimental') && !tags.includes('onexperiment'))) {
```

Comment:
I'm not sure I fully understand this migration strategy, but I also have a slight preference for `onExp` or something that really drives that this is part of our Exp platform.

Reply:
1. Allow both tag:experimental and tag:onExp settings to be ExP-assignable for now.
2. File issues for current tag:experimental users. I believe a lot of third-party extensions already use it in the non-ExP sense, meaning that the current mixup is mostly an internal or first-party issue.
3. Disallow tag:experimental settings to be ExP-assignable.